### ISSUE TITLE
Load only latest 25 Transactions in History

### ISSUE THREAD

**Issue by italocoin-project** [monero-project/monero-gui#1437]
The problem i find in history is that it stucks the whole wallet if you have a lots of transactions, my idea would be this:

Load latest 25 transactions and to se more will be added a "Load More" button

Its really needed because when i have many transactions the wallet hangs and just crashes or i need to end thask forcefully, even when syncing it would be good to not load history in the wallet because that hangs the wallet to, let me know what you guys think about it

Regards

**Comment by dEBRUYNE-1** [comment]
Kind of similar to #1433.

**Comment by erciccione** [comment]
+hacktoberfest

**Comment by NoreiT** [comment]
Would it be possible to at least not loading history after clicking history tab? I tried to find it but werent able to.

**Comment by selsta** [comment]
@NoreiT Can you rephrase that? Why should the history not load when clicking on history?

**Comment by NoreiT** [comment]
> @NoreiT Can you rephrase that? Why should the history not load when clicking on history?

Point is, that with big number of TXs, whole history is loaded right after you click History tab. And for large number of TXs, you will end up with non-responsive wallet (in better case, othewise wallet just crash after some time). 

Date filtering works only after History is loaded.

**Comment by selsta** [comment]
#2025 

+resolved
